### PR TITLE
Update node:eslint to ESLint v2

### LINKED ISSUE #225: Update to eslint 2
We'll need to update to eslint 2.

I don't think there's any rush as we need to make sure most plugins will be compatible first. But the changes seems fairly easy, so I'd say it'll be ready in a week or two.

### PR DESCRIPTION
**Pull request by zckrs**
Fix #225

### CODE REVIEW

File: generators/eslint/index.js [hunk 1]
```diff
@@ -30,12 +30,12 @@ module.exports = generators.Base.extend({
       }
     };
     var devDep = {
```
Comment (SBoudrias) [inline generators/eslint/index.js]: Can we add eslint to the devDep - I think it's a peerDeps. Also, having it here allows editor linters to work (at least with Atom and Sublime text)

Comment (zckrs) [inline generators/eslint/index.js]: Done :)

**Comment by SBoudrias** [comment]
Awesome! Can you squash your commits, then we'll be good to merge!

**Comment by zckrs** [comment]
Squashed :blush:

**Comment by SBoudrias** [comment]
Thanks!!

**Pull request merged**